Bottom: a380bfdebff0cbbcca0510a6ed13b996ed507c1f
Top:    3746609fd440ca842d2d8f4ad20c765891fc016f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 19:11:15 +0000

Refresh of hid-gtk-move-various-gdk-drawi

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 4a35d0c..f577d3e 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -46,6 +46,25 @@ Vy2 (int y)
 /* ------------------------------------------------------------ */
 
 void
+ghid_destroy_gc (hidGC gc)
+{
+  if (gc->gc)
+    g_object_unref (gc->gc);
+  g_free (gc);
+}
+
+hidGC
+ghid_make_gc (void)
+{
+  hidGC rv;
+
+  rv = g_new0 (hid_gc_struct, 1);
+  rv->me_pointer = &ghid_hid;
+  rv->colorname = Settings.BackgroundColor;
+  return rv;
+}
+
+void
 ghid_draw_grid (void)
 {
   static GdkPoint *points = 0;
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index c6a2e58..eb8ba18 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2696,27 +2696,6 @@ ghid_parse_arguments (int *argc, char ***argv)
   ghidgui->creating = TRUE;
 }
 
-
-
-void
-ghid_destroy_gc (hidGC gc)
-{
-  if (gc->gc)
-    g_object_unref (gc->gc);
-  g_free (gc);
-}
-
-hidGC
-ghid_make_gc (void)
-{
-  hidGC rv;
-
-  rv = g_new0 (hid_gc_struct, 1);
-  rv->me_pointer = &ghid_hid;
-  rv->colorname = Settings.BackgroundColor;
-  return rv;
-}
-
 void
 ghid_do_export (HID_Attr_Val * options)
 {
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index fdd7d9e..245a2c3 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -252,8 +252,6 @@ ButtonState;
 /* Function prototypes
 */
 void ghid_parse_arguments (gint * argc, gchar *** argv);
-hidGC ghid_make_gc (void);
-void ghid_destroy_gc (hidGC);
 void ghid_do_export (HID_Attr_Val * options);
 
 void ghid_create_pcb_widgets (void);
@@ -531,6 +529,8 @@ GdkPixmap *ghid_render_pixmap (int cx,
 			       int depth);
 
 /* gtkhid-gdk.c */
+hidGC ghid_make_gc (void);
+void ghid_destroy_gc (hidGC);
 void ghid_draw_grid (void);
 void ghid_draw_bg_image (void);
 void ghid_use_mask (int use_it);
